pkg/controllers/disruption/drift_test.go

```diff
@@ -151,56 +151,18 @@ var _ = Describe("Drift", func() {
 			ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeStateController, nodeClaimStateController, nodes, nodeClaims)
 			ExpectSingletonReconciled(ctx, disruptionController)
 
-			metric, found := FindMetricWithLabelValues("karpenter_nodepools_allowed_disruptions", map[string]string{
+			metric, found := FindMetricWithLabelValues("karpenter_voluntary_disruption_decisions_total", map[string]string{
```

Comment:
This is a completely different metric -- did we mean to change this? (I haven't looked at the whole test in context so this might have been wrong all along)

Reply:
We should probably actually test for both here.